packages/react-ui/src/widgets/NoDataAlert.js

```diff
@@ -1,17 +1,33 @@
 import React from 'react';
 import { Alert, AlertTitle } from '@material-ui/lab';
-import { Box } from '@material-ui/core';
+import { Box, Typography } from '@material-ui/core';
+
+function AlertBody ({ color = undefined, children }) {
+  return children ? (
+  <Box mt={0.5}>
+    <Typography variant="caption" color={color || 'inherit'} style={{ fontWeight: 'normal' }}>{children}</Typography>
+  </Box>
+) : (
+  <Box mt={-1} />
+)}
 
 function NoDataAlert({
   title = 'No data available',
-  body = 'There are no results for the combination of filters applied to your data. Try tweaking your filters, or zoom and pan the map to adjust the Map View.'
+  body = 'There are no results for the combination of filters applied to your data. Try tweaking your filters, or zoom and pan the map to adjust the Map View.',
+  severity = undefined
```

Comment:
The `severity` prop is never passed anywhere so this is removing the orange style on all existing alerts.

Reply:
AFAIK this is desired. We want to change current 'severity', to make it less prominent (not just for dropping, but for current use case).

This was already discussed with Design & validated with product.